crates/uv-warnings/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub fn disable() {
 /// Warn a user, if warnings are enabled.
 #[macro_export]
 macro_rules! warn_user {
-    ($($arg:tt)*) => {
+    ($($arg:tt)*) => {{
```

Comment:
Without the double curlies here, putting two `warn_user!` invocations back-to-back doesn't compile. My instinct was that I wanted the "warning:" prefix on both lines above, but whether or not we stick with that, this version of these macros is probably more correct.